src/firebase/snap/SubmitSnap.ts

```diff
@@ -0,0 +1,31 @@
+import Snap from './Snap'
+import firebase from 'firebase/app'
+import { SetStateAction, Dispatch } from "react";
+/* Submits a snap to the firebase collection*/
+
```

Comment:
```suggestion

/* Submits a snap to the firebase collection*/
```

The comment should go next to the method, if we're going to keep it.

Reply:
Comment is just gone altogether